joblib/_parallel_backends.py

```diff
@@ -235,6 +239,18 @@ def batch_completed(self, batch_size, duration):
                 new_duration = 0.8 * old_duration + 0.2 * duration
             self._smoothed_batch_duration = new_duration
 
+    def reset_batch_size(self):
+        """Reset batch size internal values.
+
+        This should be called after all tasks are terminated. Then future calls
+        to the same instance of a Parallel object will be performed with clean
+        initial computation statistics and won't interfere between them.
```

Comment:
This seems like a lot of words for not that much code ;-) maybe shorten it if you can. I would not insist on having the github issue in the docstring to be honest.

Maybe just move (not copy) the comment you have where reset_batch_size is called into the docstring. It seems good and short at the same time.